src/meta.js

```diff
@@ -0,0 +1,233 @@
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'apple-touch-icon.png'
+      }
+      rel='apple-touch-icon'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'apple-touch-icon-57x57.png'
+      }
+      rel='apple-touch-icon'
+      sizes='57x57'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'apple-touch-icon-60x60.png'
+      }
+      rel='apple-touch-icon'
+      sizes='60x60'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'apple-touch-icon-72x72.png'
+      }
+      rel='apple-touch-icon'
+      sizes='72x72'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'apple-touch-icon-76x76.png'
+      }
+      rel='apple-touch-icon'
+      sizes='76x76'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'apple-touch-icon-114x114.png'
+      }
+      rel='apple-touch-icon'
+      sizes='114x114'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'apple-touch-icon-120x120.png'
+      }
+      rel='apple-touch-icon'
+      sizes='120x120'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'apple-touch-icon-144x144.png'
+      }
+      rel='apple-touch-icon'
+      sizes='144x144'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'apple-touch-icon-152x152.png'
+      }
+      rel='apple-touch-icon'
+      sizes='152x152'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'apple-touch-icon-180x180.png'
+      }
+      rel='apple-touch-icon'
+      sizes='180x180'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'mstile-70x70.png'
+      }
+      rel='mstile'
+      sizes='70x70'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'mstile-144x144.png'
+      }
+      rel='mstile'
+      sizes='144x144'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'mstile-150x150.png'
+      }
+      rel='mstile'
+      sizes='150x150'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'mstile-310x150.png'
+      }
+      rel='mstile'
+      sizes='310x150'
+    />,
+    <link
+      href={
+        'https://s3.amazonaws.com/freecodecamp/favicons/' +
+        'mstile-310x310.png'
+      }
+      rel='mstile'
+      sizes='310x310'
+    />,
+    <meta
+      content='#FFFFFF'
+      name='msapplication-TileColor'
+    />,
+    <meta
+      content='/'
+      name='msapplication-TileImage'
+    />
+  ]);
```

Comment:
Almost there, each `Component` in an array requires a `key` prop. This helps react know which component is which when it comes to rendering updates. All react components are Objects, so we can asign a unique key dynamically like in this example:

```js
const arrayOfComponetents = [
  <Component1 />,
  <Component2 />,
  <Component3 />
  //...
  ]
  .map((Component, i) => ({ ...Component, key: i });
```

Where `Component` is the react component we are working with currently and `i` is it's index in the array. We return a new Object (read `Component`) with a property of `key` set to the the `Component`'s index in the array.

This will give each `Component` a unique identifier that react can use to do the voodoo they do. :sparkles: :sparkles: